collect_app/src/main/java/org/odk/collect/android/activities/AppListActivity.java

```diff
@@ -189,6 +193,11 @@ private void setupDrawerItems() {
             public View getView(int position, View convertView, ViewGroup parent) {
                 TextView textView = (TextView) super.getView(position, convertView, parent);
                 textView.setPadding(50, 0, 0, 0);
+                if ((position == 0)&&(mFirstTime)) {
```

Comment:
I would write it
`if (position == 0 && mFirstTime)`
But not everyone remembers that `==` binds tighter than `&&`, so sometimes parentheses aid readability.